### ISSUE TITLE
How to migrate from github.com/dgraph-io/badger to github.com/outcaste-io/badger?

### ISSUE THREAD

**Issue by veith** [outcaste-io/issues#1]
Is it enough to change the dependency in go.mod?

**Comment by manishrjain** [comment]
Yes, that should be sufficient. If it doesn't work, let me know.

**Comment by veith** [comment]
Sorry it doesn't work, i suggest, you have to make a release, otherwise it will try to install [v3.2103.2](https://github.com/outcaste-io/badger/releases/tag/v3.2103.2) , but this states himself as dgraph-io...

```
go: github.com/outcaste-io/badger/v3@v3.2103.2: parsing go.mod:
        module declares its path as: github.com/dgraph-io/badger/v3
                but was required as: github.com/outcaste-io/badger/v3
```

installing latest, also points to v3.2103.2

**Comment by manishrjain** [comment]
@danielmai -- Can you look into this and fix this up please?

**Comment by veith** [comment]
- **ristretto** looks good
- **dgo** points declares its path as: github.com/dgraph-io/dgo/v210

**Comment by danielmai** [comment]
BadgerDB [v3.2202.0](https://github.com/outcaste-io/badger/releases/tag/v3.2202.0) release is cut now. So the following commands can be used to do a fresh install or update:

```
go get github.com/outcaste-io/badger/v3
go get github.com/outcaste-io/badger/v3@v3.2202.0
```

**Comment by danielmai** [comment]
dgo [v210.03.1](https://github.com/outcaste-io/dgo/releases/tag/v210.03.1) release is cut now. These commands should work now for updating to the release:

```
go get github.com/outcaste-io/dgo/v210
go get github.com/outcaste-io/dgo/v210@v210.03.1
```

**Comment by veith** [comment]
I will try to install outcast-io/badger and run my tests

**Comment by danielmai** [comment]
Glad all your tests passed!

There's now a section in the docs/README on [how to migrate to outcaste-io/badger](https://github.com/outcaste-io/badger#migrating-from-dgraph-iobadger-to-outcaste-iobadger). Mainly covers updating go.mod and the Go imports.
